From f8c5fa1e6be831e8f659660b9fade5dede313956 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Fri, 9 Feb 2018 21:15:20 +0800
Subject: [PATCH] arm64: dts: rockchip: increase the i2c1 scl frequecny to 400k
 for rk3326-evb-lp3-v10

Base on the rise time of scl test, the scl frequecny could be
increased to 400k.

Change-Id: I17f858b28ed11992411c52e5f83424b0187d097c
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
index f5fd78774776..486be0962e04 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
@@ -593,6 +593,9 @@
 
 &i2c1 {
 	status = "okay";
+	clock-frequency = <400000>;
+	i2c-scl-rising-time-ns = <275>;
+	i2c-scl-falling-time-ns = <16>;
 
 	sensor@0f {
 		status = "okay";
-- 
2.35.3

